src/eos-google-chrome

```diff
@@ -94,6 +140,12 @@ class GoogleChromeLauncher:
         except OSError as e:
             exit_with_error("Could not launch Chrome: {}".format(repr(e)))
 
+        self._wait_for_installation()
```

Comment:
Yeah, I agree. I don't think this code should micro-manage what happens with the installation. Just skip this and create the file instead when the Chrome is already installed (i.e. when you run the script and you detect that Chrome is installed by e.g. checking _flatpak list | grep com.google.Chrome_ as I had mentioned to you yesterday).